Formula/zbctl.rb

```diff
@@ -0,0 +1,43 @@
+class Zbctl < Formula
+  desc "Distributed Workflow Engine for Microservices Orchestration"
+  homepage "https://docs.camunda.io/docs/apis-clients/cli-client/index/"
+  url "https://github.com/camunda-cloud/zeebe.git",
+      tag:      "1.2.9",
+      revision: "b55242898afe0dd93b87fa0b464f7c4d9a10bbad"
+  license "Apache-2.0"
```

Comment:
maybe this would be `:cannot_represent` or `all_of: ["Apache-2.0", :cannot_represent]`

> Zeebe source files are made available under the Zeebe Community License Version 1.1 except for the parts listed below, which are made available under the Apache License, Version 2.0. 

https://github.com/camunda-cloud/zeebe#license

Reply:
The `go` client files are licensed under Apache 2.0, and since that's what we're building/distributing here, wouldn't that be sufficient?

> ... except for the parts listed below, which are made available under the Apache License, Version 2.0. See individual source files for details.
>
> Available under the Apache License, Version 2.0:
> * Java Client (clients/java)
> * Go Client (clients/go)
> ...

Reply:
oh, this is only the go client, right?

yeah, if that is the case, i think it is sufficient. 👍

Reply:
I think I should correct `description` to make it more clear :)

Reply:
I've fixed the description to make it more clear.